src/applications/gi/components/vet-tec/VetTecHeadingSummary.jsx

```diff
@@ -47,12 +54,49 @@ export const VetTecHeadingSummary = ({ institution, showModal }) => {
                 (Learn more)
               </button>
             </IconWithInfo>
-            <IconWithInfo icon="map-marker" present={addressPresent}>
-              {formattedAddress}
-            </IconWithInfo>
           </div>
         </div>
       </div>
+      <div className="usa-width-two-thirds medium-8 small-12 column vads-u-margin-top--2">
+        <div className="usa-width-one-half medium-6 small-12 column">
+          <IconWithInfo icon="map-marker" present={addressPresent}>
+            {formattedAddress}
+          </IconWithInfo>
+          {/* Production flag for 19736 */}
+          {!environment.isProduction() &&
+            firstProgram && (
+              <IconWithInfo icon="globe" present={providerWebsitePresent}>
+                <a
+                  href={firstProgram.providerWebsite}
+                  target="_blank"
+                  rel="noopener noreferrer"
+                >
+                  {firstProgram.providerWebsite}
+                </a>
+              </IconWithInfo>
+            )}
+        </div>
+        {/* Production flag for 19736 */}
+        {!environment.isProduction() &&
+          firstProgram && (
```

Comment:
Is there a reason to duplicate the prod flag here and line 65?

Reply:
Just did it that way because the `IconWithInfo` for the ProviderWebsite which should be behind a production flag is contained inside of the same `div` as the `IconWithInfo` for formatted address which should not be behind a flag.